### ISSUE TITLE
Fix OutputType

### ISSUE THREAD

**Issue by ajansveld** [PaulHigin/PSThreadJob#14]
This is a trivial fix (sorry, no time to put in a PR):

```
PS> (Get-Command Start-ThreadJob).OutputType.Name
System.Management.Automation.Job2
```
Expected: ```ThreadJob.ThreadJob```

**Comment by PaulHigin** [comment]
Good point.  Easy fix.

**Comment by jazzdelightsme** [comment]
CloseMe; fixed by #17.
